OptimizelySDK/Optimizely.cs

```diff
@@ -655,5 +646,33 @@ private bool ValidateStringInputs(Dictionary<string, string> inputs)
 
             return isValid;
         }
+        
+        private object GetTypeCastedVariableValue(string value, FeatureVariable.VariableType type)
+        {
+            object result = null;
+            switch (type)
+            {
+                case FeatureVariable.VariableType.BOOLEAN:
+                    bool.TryParse(value, out bool booleanValue);
+                    result = booleanValue;
+                    break;
+                case FeatureVariable.VariableType.DOUBLE:
+                    double.TryParse(value, out double doubleValue);
```

Comment:
Does it matter that in the previous code we used the `Double` class to parse and here we are calling it from the `double` primitive type? 

`Double.TryParse(variableValue, out double doubleValue)`

Reply:
https://stackoverflow.com/questions/3951224/is-there-a-meaningful-difference-between-double-and-double-in-net
No difference.